packages/perseus-editor/src/__stories__/interactive-graph-editor.stories.tsx

```diff
@@ -0,0 +1,284 @@
+import {View} from "@khanacademy/wonder-blocks-core";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import {color, spacing} from "@khanacademy/wonder-blocks-tokens";
+import {Body, LabelLarge} from "@khanacademy/wonder-blocks-typography";
+import {action} from "@storybook/addon-actions";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import {EditorPage} from "..";
+import {
+    circleWithStartingCoordsQuestion,
+    linearSystemWithStartingCoordsQuestion,
+    linearWithStartingCoordsQuestion,
+    polygonQuestion,
+    quadraticWithStartingCoordsQuestion,
+    rayWithStartingCoordsQuestion,
+    segmentWithLockedFigures,
+    segmentWithStartingCoordsQuestion,
+    segmentsWithStartingCoordsQuestion,
+    sinusoidWithStartingCoordsQuestion,
+} from "../../../perseus/src/widgets/__testdata__/interactive-graph.testdata";
+import {registerAllWidgetsAndEditorsForTesting} from "../util/register-all-widgets-and-editors-for-testing";
+
+import EditorPageWithStorybookPreview from "./editor-page-with-storybook-preview";
+import {flags} from "./flags-for-api-options";
+
+import type {
+    DeviceType,
+    Hint,
+    PerseusAnswerArea,
+    PerseusRenderer,
+} from "@khanacademy/perseus";
+
+registerAllWidgetsAndEditorsForTesting(); // SIDE_EFFECTY!!!! :cry:
+
+export default {
+    title: "PerseusEditor/Interactive Graph",
```

Comment:
Our other widget-level editors are placed in `PerseusEditor/Widgets/*`.  Eg ["Categorizer Editor"](https://github.com/Khan/perseus/blob/be7de3a5edb9c08b6976be508c56dd4253e3c86c/packages/perseus-editor/src/widgets/__stories__/categorizer-editor.stories.tsx#L14). 


What do you think of: 
```suggestion
    title: "PerseusEditor/Widgets/Interactive Graph",
```

Reply:
I was thinking something similar! There's already [Interactive Graph Editor](https://650db21c3f5d1b2f13c02952-vxdazuuqkt.chromatic.com/?path=/docs/perseuseditor-widgets-interactive-graph-editor--docs) in that section, though.

I think my personal preference would be to put it under `PerseusEditor/EditorPage/InteractiveGraph`, since the main thing it's rendering is still the `EditorPage` component, just specifically examples of interactive graphs. That should add an "Interactive Graph" folder under the `docs` and `demo` pages, which should then contain all the interactive graph stories inside.